js/kanban.js

```diff
@@ -251,20 +322,34 @@
        * @since 9.5.0
        * @param {Object} args Object arguments
        */
-      var initParams = function(args) {
-         var overridableParams = [
+      const initParams = function(args) {
+         const overridableParams = [
             'element', 'max_team_images', 'team_image_size', 'item',
             'supported_itemtypes', 'allow_add_item', 'allow_add_column', 'dark_theme', 'background_refresh_interval',
-            'column_field', 'allow_modify_view', 'limit_addcard_columns', 'allow_order_card', 'allow_create_column'
+            'column_field', 'allow_modify_view', 'limit_addcard_columns', 'allow_order_card', 'allow_create_column',
+            'allow_delete_item'
          ];
          if (args.length === 1) {
-            for (var i = 0; i < overridableParams.length; i++) {
-               var param = overridableParams[i];
+            for (let i = 0; i < overridableParams.length; i++) {
+               const param = overridableParams[i];
                if (args[0][param] !== undefined) {
                   self[param] = args[0][param];
                }
             }
          }
+         // Set rights
+         if (args[0]['rights'] !== undefined) {
+            self.rights = new GLPIKanbanRights(args[0]['rights']);
+         } else {
+            // 9.5.0 style compatibility
+            self.rights = new GLPIKanbanRights({
+               create_item: self.allow_add_item,
+               delete_item: self.allow_delete_item,
+               create_column: self.allow_create_column,
+               modify_view: self.allow_modify_view,
+               create_card_limited_columns: self.limit_addcard_columns
```

Comment:
There is still many occurences of usage of `allow_order_card` property.
```suggestion
               create_card_limited_columns: self.limit_addcard_columns,
               order_card: self.allow_order_card
```